app/build.gradle

```diff
@@ -109,6 +109,7 @@ dependencies {
       'androidx.work:work-runtime-ktx:2.4.0',
       'com.chaos.view:pinview:1.4.3',
       'com.github.bumptech.glide:glide:4.11.0',
+      'com.github.mmin18:realtimeblurview:1.2.1',
```

Comment:
We need to check a few things before using the new library:
1. is it working correctly with Bazel?
2. Is there any license issue?
3. Do we have any other core android approach where we can make it work without any library?